docs/section22/Rule22-21.md

```diff
@@ -0,0 +1,43 @@
+
+# CHW&CW - Rule 22-21  
+
+**Rule ID:** 22-21  
+**Rule Description:** The baseline minimum condenser water reset temperature is per Table G3.1.3.11.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.11 Heat Rejection (System 7, 8, 11, 12 and 13)  
+
```

Comment:
Please align list of applicable system with the list in line 13 below for consistency

Reply:
@mkarpman I have previously asked Xing about the same and she clarified that this is the App G section reference which doesn't include the internal 'a', 'b' system nomenclature, hence not included here.